packages/block-editor/src/components/media-replace-flow/index.js

```diff
@@ -137,21 +136,26 @@ const MediaReplaceFlow = ( {
 
 	const gallery = multiple && onlyAllowsImages();
 
+	const defaultRenderToggle = ( { isOpen, onToggle } ) => (
+		<ToolbarButton
+			aria-expanded={ isOpen }
+			aria-haspopup="true"
+			onClick={ onToggle }
+			onKeyDown={ openOnArrowDown }
```

Comment:
I wonder if we should pass down these as default props for the media flow render toggle. This way, consumers don't have to re-implement a11y best practices.

Reply:
Exactly, that is what I had in mind with this [usage example](https://github.com/WordPress/gutenberg/pull/68084#pullrequestreview-2515546731). We can assume (and document) that the `renderToggle` function should accept and pass through `button` props to an eventual `button` element.

@yogeshbhutkar Let me know if you're unsure what to do here, I'd be happy to push up some proposed code.

Reply:
@mirka, yeah, reread your example after I commented and noticed it 😅